src/main/scala/com/itechart/ny_accidents/utils/DateUtils.scala

```diff
@@ -33,4 +33,9 @@ object DateUtils {
     fromLongToLocalDateTime(localDateTimeMillis)
       .withMinute(0).toInstant(ZoneOffset.UTC).toEpochMilli
   }
+
+  def getStringFromDate(date: Date, format: String): String = {
+    val dateFormat = new SimpleDateFormat(format)
```

Comment:
Use DateTimeFormatter  from java.time instead of SimpleDateFormat
DateTimeFormatter  is thread-safe, so there's no need to create one on every method call

Reply:
Done